velox/exec/tests/QueryAssertionsTest.cpp

```diff
@@ -493,4 +494,23 @@ TEST_F(QueryAssertionsTest, intervalDayTime) {
   assertQuery(plan, "SELECT * FROM tmp");
 }
 
+TEST_F(QueryAssertionsTest, equalPlans) {
+  VectorFuzzer::Options opts;
+  opts.containerVariableLength = false;
+  VectorFuzzer fuzzer(opts, pool());
+
+  auto input = fuzzer.fuzzFlat(INTEGER(), 100);
+  auto unnestPlan = PlanBuilder()
```

Comment:
No need to use unnest here. Just values(data) vs. values(data).orderBy({"c0"}) or values(data).limit(1'000'000).

Reply:
I wrote all this will relation to Unnest work so the tests carried over. But I've removed the use of unnest now.